File: java/8_kyu/yes_or_no.java
// Instruction

// Complete the method that takes a boolean value and return a "Yes" string for true, or a "No" string for false.
  
// class YesOrNo
// {
//   public static String boolToWord(boolean b)
//   {
    
//   }
  
// }


// Solution:

class YesOrNo
{
  public static String boolToWord(boolean b)
  {
    if (b){
      return "Yes";
    }else{
      return "No";
    }
  }
  
}


// Sample Tests

// import org.junit.Test;
// import static org.junit.Assert.assertEquals;
// import org.junit.runners.JUnit4;

// public class BoolTest {
//     @Test
//     public void testBoolToWord() {
//         assertEquals(YesOrNo.boolToWord(true),"Yes");
//         assertEquals(YesOrNo.boolToWord(false),"No");
//     }
// }

File: python/8_kyu/yes_or_no.py
# Instruction

# Complete the method that takes a boolean value and return a "Yes" string for true, or a "No" string for false.

# def bool_to_word(boolean):
 
  
# Solution

def bool_to_word(boolean):
    return ["No", "Yes"][boolean]
  
  
# Sample Tests

# import codewars_test as test
# from solution import bool_to_word

# @test.describe("Fixed Tests")
# def basic_tests():
#     @test.it('Basic Test Cases')
#     def basic_test_cases():
#         test.assert_equals(bool_to_word(True), 'Yes')
#         test.assert_equals(bool_to_word(False), 'No')
